content/md/what-is/what-is-an-asset.md

```diff
@@ -9,27 +9,27 @@ related: assets-transformation, upload-assets, work-with-assets
 
 # Definition of an asset
 
-An asset can be a HD image, a video, a PDF installation guide... Any kind of files that can be used to enrich a product in Akeneo.
+An asset can be an HD image, a video, a PDF installation guide... Any kind of files that can be used to enrich a product in Akeneo.
 
 ::: ee
 Assets are only available in the Enterprise Edition.
 :::
 
-An asset is always `scopable` because an asset reference file can be declined for each channel depending on your export needs.  
+An asset is always `scopable` because you made need asset reference file variations for each channel - for example, a full size image for print and a small image for ecommerce.  
```

Comment:
you may need

Reply:
```suggestion
An asset is always `scopable` because you may need asset reference file variations for each channel - for example, a full size image for print and a small image for ecommerce.  
```